mobile_config/src/boosted_hex_info.rs

```diff
@@ -128,27 +143,39 @@ impl TryFrom<BoostedHexProto> for BoostedHex {
     }
 }
 
+#[derive(Debug, Clone, Default)]
+pub struct BoostedHexes {
+    hexes: HashMap<Cell, Vec<BoostedHexInfo>>,
+}
+
 impl BoostedHexes {
-    pub fn new(hexes: Vec<BoostedHexInfo>) -> Self {
-        let hexes = hexes
-            .into_iter()
-            .map(|info| (info.location, info))
-            .collect();
-        Self { hexes }
+    pub fn test_new_active(hexes: Vec<BoostedHexInfo>) -> anyhow::Result<Self> {
```

Comment:
Unfortunately, it does not appear that it can be. 
Being defined in `mobile_config`, and depended on in `boost_manager` and `mobile_verifier`. 
`boost_manager` cannot use `#[cfg(test)]` code from another crate.

So I opted for an explicit `test_` prefix.